Gems/Vegetation/Code/Source/Components/AreaBlenderComponent.cpp

```diff
@@ -16,6 +16,8 @@
 #include <Vegetation/Ebuses/AreaSystemRequestBus.h>
 #include <Vegetation/Ebuses/DebugNotificationBus.h>
 
+#include "../VegetationProfiler.h"
```

Comment:
nit: you should have been able to just do #include<VegetationProfiler.h> here and in the other files.